core/frontend/src/ModelState.ts

```diff
@@ -268,6 +269,7 @@ export class SheetModelState extends GeometricModel2dState {
 export class SpatialModelState extends GeometricModel3dState {
   /** If this is a reality model, provides access to a list of available spatial classifiers that can be applied to it. */
   public readonly classifiers?: SpatialClassifiers;
+  public readonly scalablemeshProps?: any;

```

Comment:
I think we should enforce the type and not use "any" here if possible.

Reply:
Done! See 3f8952f